src/main/java/simplexity/simplevanish/SimpleVanish.java

```diff
@@ -39,6 +39,7 @@ public final class SimpleVanish extends JavaPlugin {
 
     private static SimpleVanish instance;
     private static boolean papiEnabled = false;
+    private static boolean pl3xmapEnabled = false;
```

Comment:
Even though pl3x is the name of the plugin, having easily complete-able and type-able names is preferable. Also it is pronounced "plex" instead of "pl-three-x".
```suggestion
    private static boolean plexmapEnabled = false;
```

This comment applies to all usages of pl3x.